spot_micro_joy/launch/everything.launch

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<launch>
+    <arg name="debug_mode" default="false"/>
+
+    <node name="i2cpwm_board_node" pkg="i2cpwm_board" type="i2cpwm_board" output="screen">
+    </node>  
+
+
+    <node name="spot_micro_motion_cmd_node" pkg="spot_micro_motion_cmd" type="spot_micro_motion_cmd_node" output="screen">
+        <rosparam command="load" file="$(find spot_micro_motion_cmd)/config/spot_micro_motion_cmd.yaml"/>
+    </node>
```

Comment:
Instead of explicitly launching these nodes, you could instead include `motion_cmd.launch` like so. 

This example also brings in the ability to override the parameter debug_mode at command line of this launch file, and pass it to the motion_cmd.launch file. 

```suggestion
<arg name="debug_mode" default="false"/>

<include file="$(find spot_micro_motion_cmd)/launch/motion_cmd.launch" >
    <arg name="debug_mode" value="$(arg debug_mode)"/>
</include>     
```

Reply:
nice, i'll take it :)